### PR TITLE
helm release fixes

### PR DESCRIPTION
**Pull request by krnowak**
Fix the RBAC rules. Update the helm package maintainers list.

### CODE REVIEW

File: helm/habitat-operator/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,12 @@ apiVersion: v1
 description: A Kubernetes controller that manages Habitat services in a Kubernetes cluster.
 engine: gotpl
 maintainers:
-  - name: Zeeshan Ali
-    email: zeeshan@kinvolk.io
+  - name: Krzesimir Nowak
+    email: krzesimir@kinvolk.io
+  - name: Iago López Galeiras
+    email: iago@okinvolk.io
```
Comment (asymmetric) [inline helm/habitat-operator/Chart.yaml]: typo

Comment (krnowak) [reply helm/habitat-operator/Chart.yaml]: Ah, so that's where this `o` went. ;)

**Comment by asymmetric** [comment]
One more case in which the manifests under `examples/` and the ones under `test/e2e/` are out of sync :(

I really think we should find a better way here.

File: helm/habitat-operator/Chart.yaml (hunk 1 above)
Comment (krnowak) [reply helm/habitat-operator/Chart.yaml]: Or should that be `iago@ocisly.io`?

**Comment by krnowak** [comment]
Updated.

**Pull request merged**